### ISSUE TITLE
Toxicity assay?

### ISSUE THREAD

**Issue by danaklug** [opensourceantibiotics/Series-2-Diarylimidazoles#3]
The original array was screened against HEK293 cells. Is there a plan for evaluating the cytotoxicity of compounds going forward? Ideally we could get this data for all analogs; barring that we should at least evaluate any compounds that would be considered for PK.

**Comment by mattodd** [comment]
It's possible we could do this internally - I've just emailed (sorry) to check. There may be alternatives here from other team members. But this is something that is crucial, so let's keep this active until we have a solution.

**Comment by danaklug** [comment]
Had a chat this morning with Alex and Ireno from Andreas Schatzlein's lab at UCL, who will be running the tox assays. A quick summary below.

- Their assay is comparable to the CO-ADD procedure and will also use HEK293 cells.

- They can screen up to 8 compounds in parallel and the turnaround time is ~3 weeks. 1-2 mg per compound is sufficient material for testing.

- We agreed that it would be a good idea to first retest a few compounds with data from CO-ADD as controls, and now need to decide on up to 5 additional compounds to be tested in the first round.

**Comment by MFernflower** [comment]
@danaklug Def need to include the compound where the bottom pyridine is swapped for phenyl

**Comment by danaklug** [comment]
Given the latest round of potency data (#35), I think we're in agreement that 869, 821, 861, and 862 should be included in the tox assay (that's the benzofuran with the phenyl, 2-, 3-, and 4-pyridyl southwest substituents). Any others?

**Comment by drc007** [comment]
Sounds good.

**Comment by mattodd** [comment]
Agreed. Original benzothiophene or, yes, 871? Maybe the active 865? And something containing the 2-pyridyl that is inactive but structurally similar, e.g. 870?

**Comment by MFernflower** [comment]
@edwintse @danaklug When you get a chance - could you compile the structures of what will be sent off for Cytotoxicity assay?

**Comment by edwintse** [comment]
The following 8 compounds will be evaluated for cytotox either late this week/early next week.

![cytotox](https://user-images.githubusercontent.com/18062981/97303495-5e18ca80-1852-11eb-9105-f96e4453c22f.jpeg)

**Comment by edwintse** [comment]
The above compounds have now been dropped off for tox screening. Will update again once results come back

**Comment by danaklug** [comment]
See below for a summary of MRC5 toxicity data for all of the NEU compounds that have been donated thus far. (Some of these compounds have tox data against other cell lines as well, but I used MRC5 for the sake of comparison.)

![NEU_Tox_1-2-2021](https://user-images.githubusercontent.com/53487492/106498561-1a52e400-6485-11eb-99d6-97c71413a203.png)

**Comment by MFernflower** [comment]
@danaklug @mattodd @drc007 

Since it appears that substituting the main core makes it more toxic - perhaps we could try isotopic replacement???!??!

![isotopic_replacement](https://user-images.githubusercontent.com/3164942/106505506-678b8180-6496-11eb-84aa-ac12ea290bdb.png)

**Comment by mattodd** [comment]
Hi @danaklug - interesting data. Not great, generally, right? 
Would you be able to make the units comparable, i.e. by converting the ug/mL to uM, and clarifying whether MIC is 50 or 90?

**Comment by danaklug** [comment]
@mattodd Yes, see below. It's a bit difficult to compare because I think we should be thinking of MIC is closer to something like an EC99 than an EC50. (From Paul: "The media I use supports more confluent growth of Staph. aureus and I use the gold standard of no visible growth in determining the MIC of compounds against this organism.") However it still doesn't look like these compounds have much of a therapeutic window.


![image](https://user-images.githubusercontent.com/53487492/106518329-55621100-649f-11eb-8ce5-71d912fe433e.png)

**Comment by MFernflower** [comment]
@danaklug @mattodd  @drc007

I guess this underscores the need to find compounds that break the potency
ceiling

**Comment by MFernflower** [comment]
@danaklug @mattodd t

It's important to note we only have MIC numbers with no MBC numbers

MBC numbers (Minimum bactericidal concentration) would be closer to an EC50 than attempting to convert MIC to EC50

**Comment by danaklug** [comment]
Alex's slides from last Friday (#56) have been uploaded [here](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Cytotox/20210219_aMRSAtox_v1.0.pptx) and she has agreed to upload her raw data as well. The table below summarizes her results from this round of screening: 

![image](https://user-images.githubusercontent.com/53487492/108981405-9c998700-7684-11eb-9466-c22dcf731ac0.png)

**Comment by danaklug** [comment]
I've handed off **OSA_979-984** to Alex for evaluation in her tox assay, with the goal of validating the promising selectivity window of **OSA_983** and giving us an idea of how the MRC5 and HEK293 results compare. Results will be posed in this Issue.
